### ISSUE TITLE
Checkout Block doesn't respect "Force shipping to the customer billing address" option.

### ISSUE THREAD

**Issue by GRSM78** [woocommerce/woocommerce-gutenberg-products-block#7219]
It seems like your Checkout Block is not respecting Shipping settings where Shipping should be forced to the billing address - even with this option on the Block allows to provide alternative address and displays Shipping address as a main one. Can you clarify how this should be working?

**Comment by senadir** [comment]
Thank you @GRSM78, I confirmed that this option is indeed not working.
When selecting the option from WooCommerce -> Settings -> Shipping -> Shipping Options
<img width="599" alt="image" src="https://user-images.githubusercontent.com/6165348/192238684-7cb99908-9efa-452a-80f0-c24590c29546.png">

You would expect Checkout to:
- Only capture a billing address.
- Base shipping fees on billing address.
As it stands, Checkout picks up both addresses and bases the shipping fees on the shipping address
<img width="697" alt="image" src="https://user-images.githubusercontent.com/6165348/192238914-a00b0234-eeb4-4f78-9e49-8208570bf129.png">
<img width="629" alt="image" src="https://user-images.githubusercontent.com/6165348/192238950-09c7aef9-8cc8-406c-8ec2-67a6aa36d887.png">

I believe this might be a recent regression because this is indeed a behavior that we implemented some time ago.
